File: src/jvmception/objects/cp/UnknownAttribute.java
package jvmception.objects.cp;

import java.io.DataInputStream;
import java.io.IOException;

public class UnknownAttribute extends FieldAttribute {

	public UnknownAttribute(String attributeName, int attributeLenght, DataInputStream dis) throws IOException {
		super(attributeName, attributeLenght, dis);
		dis.readNBytes(attributeLenght);
	}
}

File: src/jvmception/objects/cp/UnknownAttribute.cs

using System.IO;

public class UnknownAttribute : FieldAttribute
{

    public UnknownAttribute(string attributeName, int attributeLenght, BinaryReaderBigEndian dis)
        : base(attributeName, attributeLenght, dis)
    {
        dis.ReadBytes(attributeLenght);
    }
}